### PR TITLE
Pics

### PR DESCRIPTION
**Pull request by mlautman**
adding pictures to the top of tutorials and standardizing across tutorials

eg: [live](http://tutorials.picknik.ai/doc/collision_contact/collision_contact_tutorial.html)

### CODE REVIEW

File: doc/ikfast/ikfast_tutorial.rst [hunk 1]
```diff
@@ -1,6 +1,8 @@
 IKFast Plugin
 ===============================
-In this section, we will walk through configuring an IKFast plugin for MoveIt!
```
Comment (davetcoleman) [inline doc/ikfast/ikfast_tutorial.rst]: why remove this intro sentence?

File: doc/ikfast/ikfast_tutorial.rst [hunk 2]
```diff
@@ -192,7 +194,7 @@ Set the base link and EEF link to the desired index::
 
 If you have a 7 DOF arm you will need to specify a free link::
 
- export FREE_INDEX="1"
+ export FREE_INDEX="0"
```
Comment (davetcoleman) [inline doc/ikfast/ikfast_tutorial.rst]: unrelated change for this PR - why do this?

**Comment by mlautman** [comment]
@davetcoleman fixed up
